src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGateway.cs

```diff
@@ -63,19 +63,14 @@ public VirtualNetworkGateway()
         /// configurations.</param>
         /// <param name="bgpSettings">Virtual network gateway's BGP speaker
         /// settings.</param>
-        /// <param name="radiusServer">The radius server address property of
-        /// the VirtualNetworkGateway resource for vpn client
-        /// connection.</param>
-        /// <param name="radiusSecret">The radius secret property of the
-        /// VirtualNetworkGateway resource for vpn client connection.</param>
         /// <param name="resourceGuid">The resource GUID property of the
         /// VirtualNetworkGateway resource.</param>
         /// <param name="provisioningState">The provisioning state of the
         /// VirtualNetworkGateway resource. Possible values are: 'Updating',
         /// 'Deleting', and 'Failed'.</param>
         /// <param name="etag">Gets a unique read-only string that changes
         /// whenever the resource is updated.</param>
-        public VirtualNetworkGateway(string id = default(string), string name = default(string), string type = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), IList<VirtualNetworkGatewayIPConfiguration> ipConfigurations = default(IList<VirtualNetworkGatewayIPConfiguration>), string gatewayType = default(string), string vpnType = default(string), bool? enableBgp = default(bool?), bool? activeActive = default(bool?), SubResource gatewayDefaultSite = default(SubResource), VirtualNetworkGatewaySku sku = default(VirtualNetworkGatewaySku), VpnClientConfiguration vpnClientConfiguration = default(VpnClientConfiguration), BgpSettings bgpSettings = default(BgpSettings), string radiusServer = default(string), string radiusSecret = default(string), string resourceGuid = default(string), string provisioningState = default(string), string etag = default(string))
```

Comment:
This feature is not exposed to customers. It has been deployed in NRP but is protected by a feature flag. This change ensures that the contract matches NRP and the documentation that will go out to the customer once the feature is released publicly.

The correct behavior is that radius settings should go as part of VpnClientConfiguration object when the feature is released. That is also the behavior in ARM/NRP now. This is a bugfix on the previous iteration which exposed radius address.